virt/virt-4-12-release-notes.adoc

```diff
@@ -59,7 +59,7 @@ The SVVP Certification applies to:
 
 
 //CNV-19780 CNV Maturity and dataVolumeTemplates
-
+* Standalone data volumes, and data volumes created when using a `dataVolumeTemplate` to prepare a disk for a VM, are no longer stored in the system. The data volumes are now automatically collected as trash after the PVC is created. 
```

Comment:
The data volumes are now automatically collected as trash after the PVC is created. -> The data volumes are now automatically garbage collected and deleted, after the PVC is successfully populated (when the import, upload or clone is completed).